values.go

```diff
@@ -315,7 +326,13 @@ func AreEqual(left, right Value) bool {
 		if !ok {
 			return false
 		}
-		return left.AsTime().Equal(right.AsTime())
+
+		leftNano := left.AsTime().UnixNano()
+		rightNano := right.AsTime().UnixNano()
+
+		diff := abs(leftNano - rightNano)
```

Comment:
I'm totally against this. Especially if used in user formulas. A lot of usecases require greater precision than 0.1 seconds. We can just provide a round function, to round values to 0.1 seconds for example and then compare them.